main.js

```diff
@@ -59,7 +76,58 @@ function getCurrentGutter() {
 	return getGridProperties().gutter;
 }
 
+/**
+ * This sets MediaQueryListeners on all the o-grid breakpoints
+ * and fires a `o-grid.layoutChange` event on layout change.
+ */
+function enableLayoutChangeEvents() {
+	// Create a map containing all breakpoints exposed via html:before
+	const gridLayouts = getGridBreakpoints();
+	if (gridLayouts.hasOwnProperty('layouts')) {
+		const layouts = gridLayouts.layouts;
+		const breakpoints = new Map(Object.keys(layouts).map(key => [key, layouts[key]]));
```

Comment:
Please could you file an issue at https://github.com/Financial-Times/polyfill-service/ with a reduced test case. That way I can look to fix the issue and save you having to revert lots of code.